pkg/client/testclient/fixture.go

```diff
@@ -0,0 +1,192 @@
+/*
+Copyright 2014 Google Inc. All rights reserved.
```

Comment:
I see that both shippable and travis pass, but shouldnt this be 2015 instead of 2014?